src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/CustomCaConfigurationTest.java

```diff
@@ -8,6 +8,7 @@
 import com.aws.greengrass.clientdevices.auth.ClientDevicesAuthService;
 import com.aws.greengrass.clientdevices.auth.api.CertificateUpdateEvent;
 import com.aws.greengrass.clientdevices.auth.api.ClientDevicesAuthServiceApi;
+import com.aws.greengrass.clientdevices.auth.api.DomainEvents;
```

Comment:
Can you bump Nucleus back down to 2.7 and make sure this works?

Reply:
Moved it to 2.6.0 which was what we were using before all the changes, just to be safe and not change what doesn't need to